agreementbot/secure_api.go

```diff
@@ -609,7 +611,11 @@ func (a *SecureAPI) secrets(w http.ResponseWriter, r *http.Request) {
 
 		// Call the plugged in secrets provider to list the secret(s) for the input org.
 		if vaultSecretName == "" {
-			secretNames, err := a.secretProvider.ListOrgSecrets(ec.GetExchangeId(), ec.GetExchangeToken(), org)
+			userUrl := ""
+			if user != "" {
+				userUrl = fmt.Sprintf("user/%s", user)
+			}
+			secretNames, err := a.secretProvider.ListOrgSecrets(ec.GetExchangeId(), ec.GetExchangeToken(), org+cliutils.AddSlash(userUrl))
```

Comment:
A query for user private secrets should flow through the ListOrgUserSecret function in the AgbotSecrets interface, similar to how it's done when a secret name is provided.

Reply:
addressed in recent changes